frontend/components/pages/BacklogPage.vue

```diff
@@ -7,49 +7,31 @@
             </h3>
             <h2>{{ currentSprint.name }}</h2>
 
-            <table class="table table-striped mb-4">
-                <thead>
-                <tr>
-                    <th>Identifier</th>
-                    <th>Story</th>
-                    <th>Story&nbsp;points</th>
-                    <th>Status</th>
-                </tr>
-                </thead>
-                <tbody>
-                    <tr
-                        is="StoryItem"
-                        v-for="story in storiesInSprint"
-                        :key="story.id"
-                        :data="story"
-                        view="sprint"/>
-                </tbody>
-            </table>
+            <StoryTable
+                :columns="tableColumns.currentSprint"
+                :rows="storiesInSprint"
+                position-field="sprint_position"
+                view="sprint"/>
         </template>
 
-        <h2>Backlog</h2>
+        <BButton
+                :disabled="$store.state.pendingChanges < 1"
+                :title="$store.state.pendingChanges < 1 ? 'No pending changes' : 'Save all changes on page'"
+                class="float-right mb-2"
+                type="button"
+                variant="primary"
+                @click="saveAll">
+            Save all
```

Comment:
This button is only active if I edit a story. If I create a story without pressing save, the save all button is disabled.

Reply:
I've added it for adding a story - tho all this will need to be refactored again anyways with the new layout